main.go

```diff
@@ -97,7 +99,8 @@ func main() {
 				&cli.StringFlag{
 					Name:        "config-file",
 					Aliases:     []string{"f"},
-					Usage:       "the .kube/config file path",
+					Usage:       "the kubernetes config file path",
+					Value:       "~/.kube/config",
```

Comment:
It don't work :disappointed: 

Golang can't expand tilde;

```
fail to get kubernetes clientset: stat ~/.kube/config: no such file or directory
```

To do it you'll need to use one external library, maybe this one: https://github.com/mitchellh/go-homedir